src/robot_localization_estimator.cpp

```diff
@@ -182,7 +182,7 @@ void RobotLocalizationEstimator::extrapolate(
   rclcpp::Time time_stamp = rclcpp::Time(
     boundary_state.time_stamp *
     1000000000);
-  rclcpp::Duration delta_duration = rclcpp::Duration(delta * 1000000000);
+  rclcpp::Duration delta_duration = rclcpp::Duration::from_nanoseconds(delta * 1000000000);
```

Comment:
`from_seconds()` and remove the 1e9 multiplier

Reply:
Done 774841b